hera_cal/vis_clean.py

```diff
@@ -213,6 +442,46 @@ def apply_calibration(self, input_cal, unapply=False):
         apply_cal.calibrate_in_place(self.data, cal_gains, self.flags, cal_flags,
                                      gain_convention=gain_convention)
 
+    def apply_flags(self, external_flags, overwrite_flags=False, type='uvflag'):
```

Comment:
its bad to overwrite built-in python variables, so type is probably not a great choice. how about filetype? :)

Reply:
Good point. I've changed this to filetype 👍